test/common_quantization.py

```diff
@@ -120,6 +121,18 @@ def checkScriptable(self, mod, ref_inputs=None, ref_outputs=None):
         assert not ((ref_inputs is None) ^ (ref_outputs is None)), 'Both or neither' \
             'of ref_inputs and ref_outputs must be specified'
         scripted = torch.jit.script(mod)
+        self._checkScriptable(scripted)
+
+    # Call this twice: once for a scripted module and once for a traced module
+    def _checkScriptable(self, script_mod, ref_inputs=None, ref_outputs=None):
+        # Test save/load
+        #
+        # TODO: need __get_state__ and __set_state__
```

Comment:
Instead of having a `TODO` debt, we probably should create an issue or a task

Reply:
Created a tracking issue and added a link in the comment